Dockerfile

```diff
@@ -1,29 +1,30 @@
 FROM node:20-alpine AS builder
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
```

Comment:
To be testing for multi-arch

Reply:
yup looked into multi arch build before. It was working.